django/urls/resolvers.py

```diff
@@ -333,7 +334,8 @@ def resolve(self, path):
             new_path, args, kwargs = match
             # Pass any extra_kwargs as **kwargs.
             kwargs.update(self.default_args)
-            return ResolverMatch(self.callback, args, kwargs, self.pattern.name)
+            return ResolverMatch(self.callback, args, kwargs, self.pattern.name,
+                                 route=getattr(self.pattern, '_route', None))
```

Comment:
This might be able to fit on a single line after updating it, otherwise use hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). (Also, need a test that fails if this change is reverted).

Reply:
There still isn't a test that fails with this change reverted.